go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -719,23 +728,22 @@ var errJournalNotAvailable = errors.New("could not get journal for TLF")
 
 // clearConflictView tells the journal to move any pending writes elsewhere,
 // resets the CR counter, and resets the FBO to have a synced view of the TLF.
-func (fbo *folderBranchOps) clearConflictView(baseCtx context.Context) (
+func (fbo *folderBranchOps) clearConflictView(ctx context.Context) (
 	err error) {
-	// TODO later: show the cleared conflict view under a special path,
-	//  so users can copy any unmerged files manually back into
-	//  their synced view before nuking it.
+	// TODO(KBFS-3990): show the cleared conflict view under a special path,
+	//  so users can copy any unmerged files manually back into their synced
+	//  view before nuking it.
 
-	fbo.log.CDebugf(baseCtx, "Clearing conflict view")
+	fbo.log.CDebugf(ctx, "Clearing conflict view")
 	defer func() {
-		fbo.log.CDebugf(baseCtx, "Done with clearConflictView: %+v", err)
+		fbo.log.CDebugf(ctx, "Done with clearConflictView: %+v", err)
 	}()
 
 	lState := makeFBOLockState()
 	fbo.mdWriterLock.Lock(lState)
 	defer fbo.mdWriterLock.Unlock(lState)
 
-	var ctx context.Context
-	ctx = fbo.ctxWithFBOID(baseCtx)
+	ctx = fbo.ctxWithFBOID(ctx)
```

Comment:
I don't think this is needed -- the `ctx` being passed in should already have tags from where it was created (i.e. SFSID from SimpleFS).  Adding an FBOID (usually used for background tasks within `folderBranchOps`) doesn't add much and just pollutes the logs I think.

Reply:
Hmm, looks like I added this because the context needs to be replayable, but I see that it also has unintended consequences. What's a better way to get a replayable context? What is a replayable context?

Reply:
Really the context you pass in from the test should already be configured correctly.  Looks like you're using `context.Background()` in the test -- try `ctx` instead.